src/app/app.component.html

```diff
@@ -31,15 +17,20 @@
     </div>
 
     <div class="centreText centreTop catchText">
+      <div *ngIf="!casted && !fish">Cast your line!</div>
```

Comment:
Won't this have a problem later if we reset the fish or something? - Rename !fish to something else, or set a flag upon game start and check that?

Reply:
shouldn't unless you're doing something funky. possibly have a settings like "firstRun"